index.html

```diff
@@ -47,5 +47,332 @@ <h1 class="headline-1-h1">I’m Mariane
+                </div>
+            </div>
+        </section>
+        <!--works4-->
+        <section class="works-4">
+            <div class="works-4-div1">
+                <img class="works-4-div1-img" alt="" src="images/Snapshoot Portfolio4.svg">
+            </div>
+            <div class="works-4-div2">
+                <div class="works-4-div2-1">
+                    <div class="works-4-div2-1-a">
+                        <P class="works-4-div2-1-a-p">Multi-Post<br>Stories</P>
+                    </div>
+                    <div class="works-4-div2-1-b">
+                        <ul class="works-4-div2-1-b-ul">
+                            <li class="works-4-div2-1-b-ul-li-1">
+                                <span class="works-4-div2-1-b-ul-li-1-sp">CANOPY</span>
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-2">
+                                <img class="works-4-div2-1-b-ul-li-2-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-3">
+                                <span class="works-4-div2-1-b-ul-li-3-sp">Back End Dev</span>
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-4">
+                                <img class="works-4-div2-1-b-ul-li-4-img" src="images/Counter.svg" alt="">
+                            </li>
+                            <li class="works-4-div2-1-b-ul-li-5">
+                                <span class="works-4-div2-1-b-ul-li-5-sp">2015</span>
+                            </li>
+                        </ul>
+                </div>
+                <div class="works-4-div2-2">
+                    <p class="works-4-div2-2-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="works-4-div2-3">
+                    <button class="works-4-div2-3-btn1" type="button">
+                        <span class="works-4-div2-3-btn1-sp1">html</span>
+                    </button>
+                    <button class="works-4-div2-3-btn2" type="button">
+                        <span class="works-4-div2-3-btn2-sp2">css</span>
+                    </button>
+                    <button class="works-4-div2-3-btn3" type="button">
+                        <span class="works-4-div2-3-btn3-sp3">javaScript</span>
+                    </button>
+                </div>
+                <div class="works-4-div2-4">
+                    <button class="works-4-div2-4-btn" type="button">
+                        <span class="works-4-div2-4-btn-sp">See Project</span>
+                    </button>
+                </div>
+            </div>
+        </section>
+    </section>
+    <!--About Me-->
+    <section class="aboutme">
+        <div class="top">
+            <div class="top1">
+                <p class="top1-p">About<br>Myself</p>
+            </div>
```

Comment:
- [ ] The `About Myself` title represents the main title of the section and should use <hx> (h2,h3,etc...) tag , kindly fix it to be semantically correct 🙏